tools/adventure-pack/goodies/typescript/Iterator.prototype.max/index.ts

```diff
@@ -12,7 +12,7 @@ declare global {
   }
 }
 
-iteratorPrototype.max = function <T>(
+iteratorPrototype.max ??= function <T>(
```

Comment:
Can `max` exist as a built-in? I don't think I've seen it on MDN. I think we should only do this for functions that are part of some experimental proposal.

Reply:
Agreed. I was actually wondering whether there's a plan to implement a max and min built-in :) in that case we would need it. Let me go ahead and remove it